other/checkTranslationKeys.js

```diff
@@ -11,6 +11,8 @@ async function* walk(rootPath) {
 }
 
 async function checkTranslationKeys() {
+    // TODO: re-implement this
+    return;
```

Comment:
TODO - why is this affected? And should we fix this now or later? Cause later it won't end up happening

Reply:
This relied on having a separate translation.js file. I'm not 100% sure if we need this. It was mostly for the translation rework; I just left it in place because it was working. 
I could remove this and add a check into the PR template. We can do the re-implementation if we start having problems with this.

Reply:
See if you can get this to work in this PR itself.